### ISSUE TITLE
FocusOverlay color duplicated

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#151]
rgba(250,40,135,0.9) is specified in Scenery's FocusOverlay and AppendageNode's focusCircle's stroke.

Presumably color constants for accessibility should be located in one place?

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
Yes, thanks.  It is now using the color in FocusOverlay. This doesn't seem like something that should need more review.  Closing.
